<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="decompressedTB">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>480</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="sizeBeforeComprTB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="sizeAfterComprTB">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="keyLengthSP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>8</number>
    </property>
    <property name="maximum">
     <number>4096</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key length</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genNewKeyPB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate new key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>440</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompressed | WORKING ON THAT OKAY?</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>3</y>
      <width>791</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTextEdit" name="toCompressTE">
       <property name="placeholderText">
        <string>text to encrypt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="compressedTB">
       <property name="placeholderText">
        <string>compressed text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>plaintext size</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>compressed plaintext size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Suggested key size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>791</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="keyTB">
       <property name="placeholderText">
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="xorEncyptPB">
       <property name="text">
        <string>XOR ENCRYPT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="xorEncyptTB">
       <property name="placeholderText">
        <string>szyfrogram ;)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
